csp_billing_adapter_local/plugin.py

```diff
@@ -111,3 +114,79 @@ def save_csp_config(
 ):
     """Save specified content as local storage csp_config contents."""
     update_csp_config(config, csp_config, replace=True)
+
+
+@csp_billing_adapter.hookimpl(trylast=True)
+def get_usage_data(config: Config):
+    """
+    Retrieves the current usage report from the application API
+
+    :param config: The application configuration dictionary
+    :return: Return a dict with the current usage report
+    """
+    usage_data = _make_request(config.get('api'))
+    usage_data_list_name = ''.join(list(usage_data.keys()))
+
+    if usage_data_list_name in config:
+        return _extract_usage(
+            usage_data[usage_data_list_name],
+            config[usage_data_list_name]
+        )
+    if 'usage_metrics' in usage_data_list_name:
```

Comment:
Should this not be `not in` ?

Also I think there is an ordering problem and this check should be before `if usage_data_list_name in config`. As implemented I could have a config
```
- foo
  usage_metric: bar
```
and as long as `bar` will be found we's never get to exception because the `if` block prior to this test would `return` whatever `_extract_usage` returns

Reply:
> Should this not be not in ?

The check is `not in`

Reply:
@jesusbv ordering an test issue. The GitHub diff doesn't show me a reply or any code changes from when I made the original comment.

Reply:
@rjschwei that was updated in [d5d2f38](https://github.com/SUSE-Enceladus/csp-billing-adapter-local/pull/9/commits/d5d2f38145dd5fa776bb76ee771945e522e1db98)
I can squash the commits into 1, maybe that way GitHub would show the recent changes ?

Reply:
Yeah, still shows as "in" for me too and exit clause after usage.

Reply:
will squash the commits to see if that shows the right version of the code

Reply:
it shows the right version of the code for me, let me know if it's fixed for you too

Reply:
the original was OK as well
```python
if (
        'usage_metrics' in usage_data_list_name and
        usage_data_list_name in config
    ):
    ...
if 'usage_metrics' in usage_data_list_name:
        raise CSPBillingAdapterException(
            'Config missing usage metrics section'
        )
```
 means that if usage is `in` the SUMa application API response, 
 it is `not in` config, so the code was correct originally, however 
 I changed it to `not in config` as it is more clear the intention
 and it avoids confusion